### PR TITLE
New env vars and start renaming stuff

### PR DESCRIPTION
**Pull request by dluc**
New env vars for auth and start renaming microservices, UI Config to Config, and IoT Stream Analytics to Telemetry Agent

---
This change is [Reviewable](https://reviewable.io/reviews/azure/azure-iot-pcs-tools/15)

### CODE REVIEW

File: remote-monitoring/docker-compose.dotnet.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ services:
       - iothubmanager
       - devicesimulation
       - telemetry
-      - uiconfig
+      - config
```
Comment (ppathan) [inline remote-monitoring/docker-compose.dotnet.yml:15]: Do you also want to remove the circular dependency bug with this change?

File: remote-monitoring/single-vm-setup.sh [hunk 2]
```diff
@@ -35,6 +35,7 @@ export PCS_AUTH_AUDIENCE="$6"
 export PCS_WEBUI_AUTH_TYPE="aad"
 export PCS_WEBUI_AUTH_AAD_TENANT="$5"
 export PCS_WEBUI_AUTH_AAD_APPID="$6"
+export PCS_APPLICATION_SECRET=$(cat /dev/urandom | LC_CTYPE=C tr -dc 'a-zA-Z0-9-,./;:[]\(\)_=^!~' | fold -w 64 | head -n 1)
```
Comment (ppathan) [inline remote-monitoring/single-vm-setup.sh:38]: Will need to add this for k8s as well. Are you just generating a random guid?

File: remote-monitoring/docker-compose.dotnet.yml [hunk 3]
```diff
@@ -32,14 +32,20 @@ services:
...
     environment:
       - PCS_IOTHUB_CONNSTRING
-      - PCS_UICONFIG_WEBSERVICE_URL=http://uiconfig:9005/v1
+      # TODO: remove this and keep only PCS_CONFIG_WEBSERVICE_URL
+      - PCS_UICONFIG_WEBSERVICE_URL=http://config:9005/v1
+      - PCS_CONFIG_WEBSERVICE_URL=http://config:9005/v1
+      - PCS_AUTH_ISSUER
+      - PCS_AUTH_AUDIENCE
+      - PCS_APPLICATION_SECRET
```
Comment (ppathan) [inline remote-monitoring/docker-compose.dotnet.yml]: Will need to add all this new env vars to k8s .yaml as well in pcs-cli repo

**Review by ppathan — DISMISSED** [review]
LGTM

File: remote-monitoring/docker-compose.dotnet.yml (hunk 1 above)
Comment (dluc) [reply remote-monitoring/docker-compose.dotnet.yml:15]: 👍

File: remote-monitoring/docker-compose.dotnet.yml (hunk 3 above)
Comment (dluc) [reply remote-monitoring/docker-compose.dotnet.yml]: I was planning to do it, unless you prefer doing it on your side

File: remote-monitoring/single-vm-setup.sh (hunk 2 above)
Comment (dluc) [reply remote-monitoring/single-vm-setup.sh:38]: it's actually a random string of 64 chars

File: remote-monitoring/docker-compose.dotnet.yml (hunk 1 above)
Comment (dluc) [reply remote-monitoring/docker-compose.dotnet.yml:15]: done

**Pull request merged**